src/components/ListView.vue

```diff
@@ -53,21 +49,23 @@ export default {
     },
     totalItems() {
       // TODO: return total item field from this.bans
-      return 999;
+      return 10000;
```

Comment:
This doesn't solve the root issue: The server isn't telling us how many total pages there are.